dunwoody_disaster/Item.py

```diff
@@ -74,7 +74,10 @@ def to_dict(self) -> dict:
 class Weapon(Item):
     def __init__(self, name, magic, damage, magicCost, staminaCost):
         super().__init__(name)
-        self.stats = WeaponStats[name]
+        try:
+            self.stats = WeaponStats[name]
+        except KeyError:
+            self.stats = [0, 0, 0]
         self.magic = magic
         self.damage = damage
         self.magicReq = magicCost
```

Comment:
There is a disconnect between these. The `self.stats` are shown on the arsenal, but the `self.magic / self.damage / etc` are being used for the actual fight calculations. I vote we get rid of self.stats.

Reply:
I opened #57 for this issue.